pkg/apis/servicecatalog/v1alpha1/defaults.go

```diff
@@ -18,13 +18,28 @@ package v1alpha1
 
 import (
 	"github.com/satori/go.uuid"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"time"
 )
 
 func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
+func SetDefaults_ServiceBrokerSpec(spec *ServiceBrokerSpec) {
+	isValidRelistBehavior := spec.RelistBehavior == "Duration" ||
+		spec.RelistBehavior == "Manual"
+
+	if !isValidRelistBehavior {
```

Comment:
Defaults shouldn't handle invalid values, they handle missing values.
Hence, instead of the check above you should check for empty string here.